lib/hawkular/alerts/alerts_api.rb

```diff
@@ -81,12 +81,100 @@ def create_trigger(trigger, conditions = [], dampenings = [], _actions = [])
       http_post 'triggers/trigger', full_trigger
     end
 
+    # Creates the group trigger definition.
+    # @param [Trigger] trigger The group trigger to be created
+    # @return [Trigger] The newly created group trigger
+    def create_group_trigger(trigger)
+      ret = http_post 'triggers/groups', trigger.to_h
```

Comment:
Error handling is implemented at base_client.rb
I was just extending the API without touching major shared libraries, so perhaps in a major version error handling can be implemented in different way.
It happens the same with the to_h param, the json conversion is done at base_client.rb level, so the api expects to use the hash as input parameters for base http calls.